sos/report/plugins/logs.py

```diff
@@ -105,4 +105,20 @@ def postproc(self):
             r"pwd=[******]"
         )
 
+class CosLogs(Logs, CosPlugin):
+    option_list = [
+        ("log_days", "the number of days logs to collect", "", 3)
+    ]
+
+    def setup(self):
+        super(CosLogs, self).setup()
+        if self.get_option("all_logs"):
+            self.add_cmd_output("journalctl -o export")
+        else:
+            try:
+                days = int(self.get_option("log_days"))
+            except ValueError:
```

Comment:
When using `PluginOpt`, this type checking should not be needed.

Reply:
updated the file